CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+- Add AWS Custom Buckets Integration tests ([#4675](https://github.com/wazuh/wazuh-qa/pull/4675)) \- (Framework + Tests)
```

Comment:
Duplicated in line 32

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/5063/commits/772857d06a2dea4feaeba7aed697da896e071667